commands/reshowcase

```diff
@@ -102,108 +100,194 @@ const config = (() => {
   }
 })();
 
-const compiler = webpack({
-  // https://github.com/webpack/webpack-dev-server/issues/2758#issuecomment-813135032
-  // target: "web" (probably) can be removed after upgrading to webpack-dev-server v4
-  target: "web",
-  mode: isBuild ? "production" : "development",
-  entry: {
-    index: entryPath,
+const watchPlugin = {
+  name: "watchPlugin",
+  setup: (build) => {
+    build.onEnd((_buildResult) => console.log("[esbuild] Rebuild finished!"));
   },
-  output: {
-    path: outputPath,
-    filename: "reshowcase[fullhash].js",
-    globalObject: "this",
-    chunkLoadingGlobal: "reshowcase__d",
+};
+
+// entryPoint passed to htmlPlugin must be relative to the current working directory
+const entryPathRelativeToCwd = path.relative(process.cwd(), entryPath);
+
+const defaultConfig = {
+  entryPoints: [entryPath],
+  entryNames: "[name]-[hash]",
+  assetNames: "[name]-[hash]",
+  chunkNames: "[name]-[hash]",
```

Comment:
These are needed because otherwise it would use the [default](https://github.com/evanw/esbuild/blob/332727499e62315cff4ecaff9fa8b86336555e46/cmd/esbuild/main.go#L74) `[dir]/[name]` right? Not sure if worth a comment.

Reply:
I actually do not no, they were here before me. Will check